src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -883,7 +883,7 @@ Use "{{.CommandPath}} [command] --help" for more information about a command.{{e
 		t := template.New("top")
 		t.Funcs(templateFuncs)
 		template.Must(t.Parse(text))
-		return t.Execute(cmd.Out(), cmd)
+		return t.Execute(cmd.OutOrStderr(), cmd)
 		return originalUsageFunc(cmd)
```

Comment:
LGTM, although this line looks like dead code (probably my fault) - should be safe to remove, but I'm surprised this doesn't trigger lint.